setup.go

```diff
@@ -340,6 +350,12 @@ func configFromEnv() Config {
 		}
 	}
 
+	if noExplicitFormat &&
+		(!cfg.Stdout || !isTerm(os.Stdout)) &&
+		(!cfg.Stderr || !isTerm(os.Stderr)) {
```

Comment:
post-merge nit: `!(cfg.Stdout && isTerm(os.Stdout)` is significantly easier to read.

Alternatively, I _think_ we could have just checked the output file. If the user _decides_ they want to write to `/dev/ttyN`. Otherwise, how _else_ am I going to send my logs to tty9?

Reply:
We actually need both - please see https://github.com/ipfs/go-log/pull/117